packages/sdk/src/types/TransactionExpiry.ts

```diff
@@ -28,9 +27,31 @@ class TransactionExpiry {
         public readonly expiryEpochSeconds: bigint
     ) {}
 
-    public toJSON(): number {
-        return Number(this.expiryEpochSeconds);
+    /**
+     * Get a string representation of the transaction expiry date in seconds since the Unix epoch.
+     * @returns {string} The string representation.
+     */
+    public toString(): string {
+        return this.expiryEpochSeconds.toString();
     }
+
+    /**
+     * Get a JSON-serializable representation of the transaction expiry date.
+     * @throws If the expiry is not representable as a number.
+     * @returns {bigint} The JSON-serializable representation.
+     */
+    public toJSON(): bigint {
+        return this.expiryEpochSeconds;
+    }
```

Comment:
I think we should declare this change in the changelog as a bug fix. It's a breaking change, but it was wrong before.

Reply:
The reason why this is OK('ish), is because the normal fix for this is to `JSONBig.stringify(...)`, which will give the number representation without losing precision, i.e. the deserializing party will get the same value.

In the case of changing the representation of `CcdAmount` from `string` to `bigint`, `JSONBig.stringify` will produce a `number` instead of `string`, causing deserialization to fail. An example of where this will fail is mobile wallets.